intel_extension_for_transformers/neural_chat/requirements.txt

```diff
@@ -33,3 +33,4 @@ sentence_transformers
 unstructured
 markdown
 rouge_score
+openpyxl
```

Comment:
please confirm with Wenxin whether it has been added BOM

Reply:
> please confirm with Wenxin whether it has been added BOM

I left a message to Wenxin about this package. It's license is MIT license. Should be ok